students/11th/kungyulee/westagram/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import path, include
+
+urlpatterns = [
+    path('', include('accounts.urls')),
```

Comment:
account 지정해주는게 명확하고 좋을것 같습니다. 왜냐하면 나중에 앱들이 추가되면 이에대한 구분이 필요하기 때문입니다!
```suggestion
    path('accounts', include('accounts.urls')),
```